### ISSUE TITLE
{question_link} does not seem to work in email templates.

### ISSUE THREAD

**Issue by nextgenthemes** [anspress/anspress#530]
Version 4.1.17 from wp.org

This is my email template:

```
A new question is posted by {asker}

<a href="{question_link}">{question_title}</a>
{question_content}

Please do respond on the support site and do not reply to this automated email.
```

This is what I get in emails.
```html
...
<a href="%7Bquestion_link%7D">...</a>
...
```

Everything else works but just not the links the questions.

**Comment by rahularyan** [comment]
Hello @nextgenthemes 
I am not able to replicate this issue with our latest version.
But its clearly visible why its happening. `{` is converted to `%7B` Please fix that by switching to text mode while editing email template.

**Comment by nextgenthemes** [comment]
@rahularyan I already checked the text editor and in fact copied it from there into GitHub. There are is %7B in there.

**Comment by rahularyan** [comment]
Have your tried like this:
![Annotation 2020-04-24 091856](https://user-images.githubusercontent.com/916764/80173206-b5708080-860c-11ea-8575-4fbefbb7c7e5.jpg)
If this does not help please try to update `%7B` to `{` in email templates from database.

**Comment by nextgenthemes** [comment]
Thanks for your response.

I am not sure what you are trying to show me with this screenshot and your red lines but I just manually retyped relevant part back in it is still not fixed. I think about the some filter or `esc_url` function or something like that. It is actually like this, I am not sure why I did not show it like this above but:

`<a href="http://%7Bquestion_link%7D" rel="noreferrer nofollow noopener">...</a>`

There is a `http://` first and then the `%7B`. Looks like the `esc_url` function is run on the string that never for replaced with the url for some reason. That also encodes the brackets. And as I have guessed it is stored correctly in the DB.
 
```
question_excerpt} Link: {question_link}\";s:24:\"new_answer_ema
ited by {editor}. Link: {question_link}\";s:25:\"edit_answer_em
ited by {editor}. Link: {question_link}\";s:23:\"categories_ima
y {editor}\n\n<a href=\"{question_link}\">{question_title}</a>\
by {asker}\n\n<a href=\"{question_link}\">{question_title}</a>\
```

I just tried to use {question_link} on its own it is working. So its only when inside `href=""`
